### ISSUE TITLE
Run tests on the entire DOM. Not only the react part?

### ISSUE THREAD

**Issue by badtant** [dylanb/react-axe#4]
Hi,

I'm trying out react-axe which seems quite nice. Though I'm not getting some if the errors I expect.
For example "Ensures every HTML document has a lang attribute". I'm getting it through the axe chrome plugin and also if I'm running axe-core with the following:

```
var axe = require("axe-core/axe.js");

axe.a11yCheck(document, function (results) {
console.log(results.violations);
});
```

Is it possible to get it also with react-axe? I guess maybe not since react is only handling a part of the DOM and not html and head for example...

**Comment by badtant** [comment]
Digging a litte i changed:
`checkAndReport(document.body, timeout);`

to
`checkAndReport(document, timeout);`

which seems to work fine.

**Comment by dylanb** [comment]
@badtant  you want to submit a PR?

**Comment by dylanb** [comment]
published

**Comment by badtant** [comment]
Doesn't seem like this was included in 1.0.7.
Probably just something wrong with the merge. Would be great with a 1.0.8 with this included.

**Comment by badtant** [comment]
Ahh, it was because I switched it back here to separate each PR.
https://github.com/dylanb/react-axe/commit/952f13bc4d260f95ea7fe058c47eb950cd682888

Do you want to fix it or should I make a new PR?
